service/nginx/nginx.conf

```diff
@@ -21,15 +21,22 @@ http {
         server 127.0.0.1:9000;
     }
 
+    index index.html;
+
+
     server {
         listen  8080;
         server_name yo;
 
 
+
```

Comment:
do not serve anything but json-rpc, the production service should not be serving docs